### PR TITLE
setting page has tabs for categories, yeh!

### PR DESCRIPTION
**Pull request by abenari**

### CODE REVIEW

File: app/views/settings/index.html.erb [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
+              <td><%=h item.description %></td>
+            </tr>
+          <% end %>
+        </table>
+      </div>
   <% end %>
-</table>
+</div>
 
-<%= page_entries_info @settings %>
```
Comment (skottler) [inline app/views/settings/index.html.erb:20]: This seems reasonable, but just want to confirm that pagination won't be around any longer now that there are categories?

File: app/controllers/settings_controller.rb [hunk 2]
```diff
@@ -3,12 +3,11 @@ class SettingsController < ApplicationController
   before_filter :require_admin
 
   def index
-    values = Setting.search_for(params[:search], :order => params[:order])
+    @settings = Setting.search_for(params[:search], :order => params[:order])
```
Comment (ohadlevy) [inline app/controllers/settings_controller.rb]: i guess you dont need order?

File: app/views/settings/index.html.erb (hunk 1 above)
Comment (abenari) [inline app/views/settings/index.html.erb:20]: Yep, lists are shorter now no need for pagination.

**Comment by skottler** [comment]
This got merged earlier today. Closing it out.

**Pull request closed without merging**